pipelines/folioRancher/folioNamespaceTools/manageNamespace/createNamespaceFromBranch/Jenkinsfile

```diff
@@ -1,4 +1,6 @@
 #!groovy
+package folioRancher.folioNamespaceTools.manageNamespace.createNamespaceFromBranch
```

Comment:
Please avoid this addings from idea, this is not reqired change

Reply:
Sure Boss, thanks for the suggestion!